packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -172,7 +172,7 @@ export interface CommonCallingHandlers {
     onLowerHand: () => Promise<void>;
     // (undocumented)
     onRaiseHand: () => Promise<void>;
-    // (undocumented)
+    // @beta (undocumented)
```

Comment:
Should we have some JS doc on the interface for the handlers on this?

Reply:
None of the handlers have the js-doc here. So I left it undocumented as well. And its pretty intuitive as the handlers are self-explanatory for the feature with other similar APIs documented accross the package.